src/Stratis.Bitcoin.Features.Miner.Tests/PosMintingTest.cs

```diff
@@ -525,11 +525,11 @@ public void CoinstakeAge_PrevOutIsCoinstake_BeforeActivation_Mainnet()
         [Fact]
         public void CoinstakeAge_PrevOutIsCoinstake_AfterActivation_Mainnet()
         {
-            int activationHeight = PosConsensusOptions.CoinstakeMinConfirmationActivationHeightMainnet;
-            int afterActivationHeight = activationHeight + 1000;
-
-            Assert.True(this.WasUtxoSelectedForStaking(KnownNetworks.StratisMain, afterActivationHeight, afterActivationHeight - 498, true));
-            Assert.False(this.WasUtxoSelectedForStaking(KnownNetworks.StratisMain, afterActivationHeight, afterActivationHeight - 497, true));
+            // The logic here is that, before the activation, a coinstake UTXO requires 50 confirmations on mainnet to be used as a staking candidate.
```

Comment:
This test looks screwy, like it's been swapped around with one of the others. I'll check.

Reply:
@zeptin in 307 it looks like this?

https://github.com/stratisproject/StratisBitcoinFullNode/blob/884dbae65c46a48db9da5a82d758eecf106f7584/src/Stratis.Bitcoin.Features.Miner.Tests/PosMintingTest.cs

Reply:
Will fix